yaml/morphology/extension.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 name: extension
-brief: transcending vs. remains within the discursive context; spatiotemporal part and dynamics
+brief: spatiotemporal limits relative to current context
```

Comment:
I was under the impression it was limits wrt the matter under discussion. Otherwise wouldn't we have to use PRX *always* when talking about things not physically present/accessible?

Reply:
Yes, I think I see why "spatiotemporal limits relative to current context" isn't great. Should we just leave it at "spatiotemporal part and dynamics"? "Spatiotemporal limit and dynamics"?